app/models/miq_provision_workflow.rb

```diff
@@ -3,17 +3,17 @@ def self.base_model
     MiqProvisionWorkflow
   end
 
+  # Find the appropriate workflow class for the given +platform+ string. For
+  # example, the argument "openstack" would become:
+  #
+  #  "ManageIQ::Providers::Openstack::CloudManager::ProvisionWorkflow"
+  #
   def self.class_for_platform(platform)
```

Comment:
@Fryguy Are we ready to drop support for the old naming scheme?

Reply:
@agrare Do you know of any providers that still use these old constant names? (`OpenstackInfra`)

Reply:
I'm not really sure... at a minimum @djberg96 can you do cross-repo tests on this to ensure the constant is not being used anywhere?

Reply:
@Fryguy https://github.com/ManageIQ/manageiq-cross_repo-tests/pull/55

Reply:
We should probably log a deprecation warning for a release first.  What do you think?

Reply:
@bdunne I guess I'm not following how you want it to look then.

Reply:
```
if classy =~ /(.*)Infra/
  Vmdb::Deprecation.deprecation_warning("MiqProvisionWorkflow.class_for_platform with platform #{platform}")
  find_matching_constant("MiqProvision#{classy}Workflow") ||
...
```

would give you 
```
DEPRECATION WARNING: MiqProvisionWorkflow.class_for_platform with platform OpenstackInfra is deprecated and will be removed from ManageIQ L-release (called from irb_binding at (irb):4)
```

Reply:
I guess I don't see the benefit of that. The cross-repo tests passed, and nothing uses the old naming scheme any more, right?

Reply:
> I guess I don't see the benefit of that. The cross-repo tests passed, and nothing uses the old naming scheme any more, right?

Yes, the code paths that have test coverage aren't using it.

@gmcculloug What's your comfort level with this?

Reply:
@bdunne Made the change locally and tried a few paths, all looked good.  👍